addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -419,6 +583,53 @@ function ThreadFactory({ Entity }) {
             await this.async(() => currentPartnerFollower.remove());
         }
 
+        /**
+         * Called when current partner has explicitly stopped inserting some
+         * input in composer. Useful to notify current partner has currently
+         * stopped typing something in the composer of this thread to all other
+         * members.
+         *
+         * @param {Object} [param0={}]
+         * @param {boolean} [param0.immediateNotify=false] if set, is typing
+         *   status of current partner is immediately notified and doesn't
+         *   consume throttling at all.
+         */
+        async unregisterCurrentPartnerIsTyping({ immediateNotify = false } = {}) {
```

Comment:
This method is only called with true

Reply:
just post message should have it set to `true`, changed that